features/step_definitions/restaurant_steps.rb

```diff
@@ -105,6 +105,13 @@
   login_as(user, scope: :user)
 end
 
+Given(/^"([^"]*)" is a highlighted dish$/) do |dish_name|
+  dish = Dish.find_by(name: dish_name)
+  dish.highlight = true
+  dish.save
+  expect(dish.highlight).to eq true
```

Comment:
Please don't add assertions in steps that are setting the stage for the test ('Given' steps)